File: BaseLib/src/main/java/com/hjc/baselib/utils/permission/PermissionCallBack.java
package com.hjc.baselib.utils.permission;

/**
 * @Author: HJC
 * @Date: 2019/7/29 14:28
 * @Description: AndPermission回调监听
 */
public interface PermissionCallBack {
    void onGranted();

    void onDenied();
}

File: BaseLib/src/main/java/com/hjc/baselib/utils/permission/PermissionCallBack.kt
package com.hjc.baselib.utils.permission

/**
 * @Author: HJC
 * @Date: 2020/1/3 18:06
 * @Description: AndPermission回调监听
 */
interface PermissionCallBack {
    fun onGranted()
    fun onDenied()
}